client/js/components/procedure/DpSearchProcedures.vue

```diff
@@ -19,7 +19,6 @@
       }"
       @enter="search" />
     <dp-button
-      class="ml-0.5 mt-0.5"
```

Comment:
I thought, we want to define outer spacings outside the component. Why did You remove it?

Reply:
right! while the top margin led to some weird layout, the left margin is still needed. re-added it in 4484babb146eaa5b4a8857555a6173bf08300933